docs/experimental/nnx/mnist_tutorial.md

```diff
@@ -170,18 +174,23 @@ This function takes the `state` and a data `batch` and does the following:
 
 ```{code-cell} ipython3
 @jax.jit
-def train_step(state: TrainState, batch):
+def train_step(state: nnx.State, static: nnx.GraphDef, batch):
```

Comment:
maybe we could use `nnx.jit` here to make it simpler?

Reply:
Done.

Should we show some examples elsewhere on how to use NNX with both `nnx.jit` and `jax.jit`; in toy examples, and/or a guide with a side-by-side comparison, similar to the upgrade guides?